Cargo.toml

```diff
@@ -26,26 +24,26 @@ serde_json = { version = "1.0", optional = true }
 ordered-float = "2.0"
 warp10 = { version = "2.0.0", optional = true }
 rand = { version = "0.7.3" }
-time = "0.3.20"
-colored = "2.0.0"
-chrono = "0.4.19"
+time = "0.3"
+colored = "2.0"
+chrono = "0.4"
```

Comment:
what is the rationale for not pointing to specific versions ?

Reply:
it allows to update automatically to the last bugfix release